tensorflow/core/kernels/eigen_spatial_convolutions_test.cc

```diff
@@ -1605,6 +1605,7 @@ static void PackLhsHelper(int iters,
       /*inner_dim_reordered*/ false,                  //
       /*Alignment*/ 0>;
 
+  using Traits = typename Eigen::internal::gebp_traits<float, float>;
```

Comment:
Can you move Trains under the ifdef where it's required.

Reply:
Done.